agenta-cli/agenta/cli/helper.py

```diff
@@ -83,16 +83,18 @@ def get_api_key() -> str:
             return api_key
         elif confirm_api_key is None:  # User pressed Ctrl+C
```

Comment:
this should be modified so that the user can answer by Y or n, and if they answer no, then we go to the next logic where they can provide a new key

Reply:
@mmabrouk That's what it does. 

<img width="1180" alt="image" src="https://github.com/Agenta-AI/agenta/assets/56418363/16c5186c-74aa-42fa-ac5e-cb5df47b709e">

Reply:
aa thanks!